core/drivers/clk/clk-stm32mp13.c

```diff
@@ -1325,7 +1325,9 @@ static int clk_stm32_pll_init(struct clk_stm32_priv *priv, int pll_idx,
 				     RCC_PLLNCR_DIVPEN | RCC_PLLNCR_DIVQEN |
 				     RCC_PLLNCR_DIVREN);
 
-			clk_stm32_gate_rdy_disable(pll->gate_id);
+			if (clk_stm32_gate_rdy_disable(pll->gate_id)
+			    != TEE_SUCCESS)
```

Comment:
You could discard `!= TEE_SUCCESS`, implicit by GPD TEE spec.
```c
			if (clk_stm32_gate_rdy_disable(pll->gate_id))
				return -1;

Reply:
Ok, I will fixed the others too